packages/toast/src/ProgressBar/ProgressBar.tsx

```diff
@@ -59,7 +57,7 @@ const progressBarThemeStyle: Record<Theme, string> = {
       #083c90 ${toastWidth}px
     );
   `,
-  [Theme.Light]: css`
+  [Theme.Dark]: css`
     background-color: #0498ec;
     background-image: linear-gradient(
       90deg,
```

Comment:
Let's move this to a `ProgressBar.styles.ts` if we want to call the restructuring done